### ISSUE TITLE
PyCharm, PhpStorm, etc

### ISSUE THREAD

**Issue by scaba** [lra/mackup#949]
Is it possible to add using wildcards for the JetBrains applications? Their config file naming includes the app's version, so every time a new version is released, I have to either manually create the links or wait until Mackup releases a new version with the new JetBrains app support.

**Comment by derimagia** [comment]
Agreed, I've been wanting to use mackup so I can remove it from my dotfiles. Wildcard would be pretty critical because of the config-folders-per-versions that Jetbrain uses.

**Comment by tobymccann** [comment]
This would be great

**Comment by webberwang** [comment]
+1

**Comment by lauragra-y** [comment]
7 years on - this would still be nice! PyCharm versions change so often, having them harcoded is a headache.
